### PR TITLE
Fixes docs for DecomposedDataUri

### PR DESCRIPTION
**Pull request by Zyie**
##### Description of change

This pr fixes the automatically generated types where `DecomposedDataUri` is not exported. This has been an issue before in #6150 where interfaces are stripped out along with there comments.

To document it in a way that it doesn't get removed means defining it below its declaration
##### Pre-Merge Checklist

- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by themoonrat** [comment]
Feels odd to have the documentation separate to the code :/ Is this a temp thing until we're fully .ts and find a different solution for docs?

**Comment by Zyie** [comment]
Yes i believe this is temporary until the conversion is complete. Then we will remove jsdoc for something that works better with typescript.

File: packages/utils/src/network/decomposeDataUri.ts [hunk 1]
```diff
@@ -1,31 +1,47 @@
...
     encoding: string;
-    /**
-     * @property {string} data The actual data
-     */
     data: string;
 }
 
+/**
+ * @memberof PIXI.utils
+ * @interface DecomposedDataUri
```
Comment (bigtimebuddy) [inline packages/utils/src/network/decomposeDataUri.ts]: Why not just add the properties here?

Comment (Zyie) [reply packages/utils/src/network/decomposeDataUri.ts]: I'm not sure what you mean? I tried adding member like so but this didnt work

```
/**
 * @memberof PIXI.utils
 * @interface DecomposedDataUri
 * @member {string} mediaType type, eg. `image`
 * @member {string} subType Sub type, eg. `png`
 * @member {string} charset type, eg. `image`
 * @member {string} encoding Data encoding, eg. `base64`
 * @member {string} data The actual data
 */
```

I just copied the structure that IPoint implemented for its docs

Comment (bigtimebuddy) [reply packages/utils/src/network/decomposeDataUri.ts]: Try `@property`

Comment (Zyie) [reply packages/utils/src/network/decomposeDataUri.ts]: Adding property results in an empty interface

```
/**
 * @memberof PIXI.utils
 * @interface DecomposedDataUri
 * @property {string} mediaType type, eg. `image`
 * @property {string} subType Sub type, eg. `png`
 * @property {string} charset type, eg. `image`
 * @property {string} encoding Data encoding, eg. `base64`
 * @property {string} data The actual data
 */
interface DecomposedDataUri {
}
```

Comment (bigtimebuddy) [reply packages/utils/src/network/decomposeDataUri.ts]: Okay. Not use fighting with tsd jsdocs

**Pull request merged**